client/dogs/FormPage.js

```diff
@@ -316,6 +318,8 @@ type State = {
   rabiesIssued: string,
   rabiesExpire: string,
   comments: string,
+  rabiesCert: Array<Object>,
```

Comment:
Bookmark this page: https://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/

It is a list of all of Flow's built-in types, which include DOM, Node, and other types.

I am pretty sure that you want to adjust this type to `Array<File>`, but double-check (by `console.log`ging `typeof file` in the `onDrop` method) that `File` is what `react-dropzone` is giving you.